falcon/request.py

```diff
@@ -683,6 +692,14 @@ def access_route(self):
     def remote_addr(self):
         return self.env.get('REMOTE_ADDR')
 
+    @property
+    def port(self):
+        return self.env.get('SERVER_PORT')
+
+    @property
+    def netloc(self):
+        return self.host + ':' + self.port
```

Comment:
Usually the port will be derived from `HTTP_HOST` and may be equal to `None` if the client did not specify one explicitly (depending on how we handle that case).